ci/filter.nix

```diff
@@ -113,6 +113,7 @@ let
     "ocaml-migrate-parsetree"
     "wasm"
     "pythonlib"
+    "camlp5"
```

Comment:
camlp5 seems to not be available for OCaml >= 4.13?

Reply:
sure does https://github.com/anmonteiro/nix-overlays/blob/master/ocaml/default.nix#L171